src/PAModel/PAConvert/Yaml/YamlWriter.cs

```diff
@@ -61,6 +62,9 @@ public void WriteProperty(string propertyName, string value)
                 return;
             }
 
+            // Normalize newlines in value
+            value = value.Replace("\r\n", "\n").Replace("\r", "\n");
```

Comment:
Is there a test we can add for this? 
This is significant change - means we don't round trip exactly, although our checksum is probably flexible enough.